### PR TITLE
Ensure order entity matches the full order entity spec

### PR DESCRIPTION
**Pull request by probably-not**

### CODE REVIEW

**Comment by probably-not** [comment]
This will close #211

**Comment by probably-not** [comment]
I haven't checked any of the other entities (I'm working mostly with the Orders entity right now) but I can do a quick review to make sure those match according to the spec as well

File: alpaca/entities.go [hunk 1]
```diff
@@ -52,12 +52,13 @@ type Order struct {
...
 	AssetID        string           `json:"asset_id"`
 	Symbol         string           `json:"symbol"`
-	Exchange       string           `json:"exchange"`
 	Class          string           `json:"asset_class"`
+	OrderClass     OrderClass       `json:"order_class"`
 	Qty            *decimal.Decimal `json:"qty"`
 	Notional       *decimal.Decimal `json:"notional"`
 	FilledQty      decimal.Decimal  `json:"filled_qty"`
-	Type           OrderType        `json:"order_type"`
+	OrderType      OrderType        `json:"order_type"`
```
Comment (gnvk) [inline alpaca/entities.go:60]: `order_type` is deprecated, so it shouldn't be added, you can simply use `type` instead.

**Review by gnvk — APPROVED** [review]
LGTM. Thank you for your contribution.

**Pull request merged**